lib/data/conversation/search.ts

```
@@ -186,6 +187,19 @@ export const searchConversations = async (
         ) as recent_message`,
         sql`true`,
       )
+      .leftJoin(
+        sql`LATERAL (
+          SELECT COUNT(*) as count
+          FROM ${conversationMessages}
+          WHERE ${and(
+            eq(conversationMessages.conversationId, conversations.id),
+            eq(conversationMessages.role, "user"),
+            isNull(conversationMessages.deletedAt),
+            gt(conversationMessages.createdAt, sql`COALESCE(${conversations.lastReadAt}, ${conversations.createdAt})`)
```

ESLint

[error] Insert `,`

(prettier/prettier)